src/applications/claims-status/components/AppealListItem.jsx

```diff
@@ -1,54 +0,0 @@
-import _ from 'lodash';
-import { Link } from 'react-router';
-import moment from 'moment';
-import PropTypes from 'prop-types';
-import React from 'react';
-
-import { appealStatusDescriptions } from '../utils/appeal-helpers.jsx';
-
-const renderNextAction = (lastEvent, previousHistory) => {
-  if (lastEvent.type === 'ssoc' && previousHistory[0].type !== 'soc') {
-    return null;
-  }
-  const nextAction = appealStatusDescriptions(lastEvent, previousHistory)
-    .nextAction;
-  return nextAction && nextAction.title;
-};
-
-export default function AppealListItem({ appeal }) {
-  // always show merged event on top
-  const events = _.orderBy(
-    appeal.attributes.events,
-    [e => e.type === 'merged', e => moment(e.date).unix()],
-    ['desc', 'desc'],
-  );
-  const lastEvent = events[0];
-  const firstEvent = events[events.length - 1];
-  const previousHistory = events.slice(1);
-
-  return (
-    <div className="claim-list-item-container">
-      <div className="last-changed-label">
```

Comment:
Same with `last-changed-label`

![Ripgrep search on "last-changed-label" with the only result being in a CSS file](https://user-images.githubusercontent.com/2008881/194413504-aa2e1cc8-3efd-4d70-820d-df294c1946d9.png)